bf2fc6cc711aee1a0c2a/kas-fleet-manager
# chore: add registered_users_per_organisation parameter to deploy/service make target

## Description
Adding `REGISTERED_USERS_PER_ORANISATION` parameter to the `deploy/service` make target in cases where the default value needs to be modified.

## Verification Steps
1. Run make deploy/service and ensure that the value read from `REGISTERED_USERS_PER_ORGANISATION` is read and set in the `quota-management-configuration.yaml` of the `kas-fleet-manager-allowed-users-config` configmap. 

## Checklist (Definition of Done)
